### ISSUE TITLE
;;lockdown command

### ISSUE THREAD

**Issue by RealKC** [Frederikam/FredBoat#220]
This would be a moderation command for the main boat, requiring `MANAGE_CHANNELS` permission for both the invoker and, of course, the bot.

Usage: 
`;;lockdown <channel> <duration> <message>` or
`;;lockdown <duration> <message>` if you're using the command in the channel you wish to lockdown.

Maybe if the user doesn't specify a duration the bot should default to 30 mins?

Also `;;lockdown <channel> off` should unlock the channel.

Maybe a way to make it so specific roles don't get locked out of the channel when the command is used, maybe some ;;config thing?

**Comment by Nanabell** [comment]
i disagree with a default time, if no time is provided it should have no time at all and just stay until its unlocked manually or with the command, also disagree with the config idea we should only touch the `@everyone` Role not any other if you want to permit certain users manually grant them access, if we changed every setting we would need to store the channel settings in a db/memory which could cause problems if a role was deleted in the timeframe.

I assume you mean with [message] that it auto posts a message in the channel but i dont know if it wouldnt be better for the staff to post the message not a "random" bot  

Also about putting this into the "main" bot, since we want to merge the main bot with the music one i think we should not put any new developments into it rather put them into the music one which will ultimately be the only remaining

Feedback ?

**Comment by knight-ryu12** [comment]
I'll disagree.
thus, locking down will confuse other bots.
